cli/viam/main.go

```diff
@@ -905,6 +909,46 @@ viam module upload --version "0.1.0" --platform "linux/amd64" packaged-module.ta
 					},
 				},
 			},
+			{
+				Name:  "version",
+				Usage: "print version info for this program",
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:  "debug",
+						Usage: "dump full BuildInfo",
+					},
+				},
+				Action: func(c *cli.Context) error {
+					if info, ok := debug.ReadBuildInfo(); !ok {
+						log.Fatal("Error reading build info")
```

Comment:
[nit] can you return the error here instead of fatal-logging? It'll get propagated up to the user. I'll try to add better error handling in RSDK-4076. This might not relevant after cheuk's comment, but goes for any error.